ONSdigital/eq-questionnaire-runner
# Add translations for visually hidden error message

### What is the context of this PR?
Adds a missing translation.

### How to review 
- Run `make translate`
- Launch `test_radio_mandatory` while setting launch language to `cy` and ensure the error message is translated when you submit without choosing an answer.


### Checklist
